ros2_realtime_examples/package.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>ros2_realtime_examples</name>
+  <version>0.0.1</version>
+  <description>ros_realtime_examples metapackage</description>
+  <maintainer email="carlos.svic@gmail.com">Carlos San Vicente</maintainer>
+  <license>Apache License 2.0</license>
+  <author>Carlos San Vicente</author>
+
+  <buildtool_depend>ament_cmake</buildtool_depend>
+
+  <build_depend>minimal_memory_lock</build_depend>
```

Comment:
@carlossvg, the plan here is to increase this meta-package with each package?

Reply:
Exaclty